paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -0,0 +1,56 @@
+package com.stripe.android.customersheet.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.navigation.PaymentSheetScreen
+import com.stripe.android.paymentsheet.ui.PaymentSheetScaffold
+import com.stripe.android.paymentsheet.ui.PaymentSheetTopBar
+import com.stripe.android.ui.core.elements.H4Text
+
+@Composable
+internal fun CustomerSheetScreen(
+    header: String?,
+    isLiveMode: Boolean,
+    isProcessing: Boolean,
+    isEditing: Boolean,
+    onBackPressed: () -> Unit = {},
+    onEdit: () -> Unit = {}
+) {
+    val horizontalPadding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
+    val bottomPadding = dimensionResource(R.dimen.stripe_paymentsheet_button_container_spacing_bottom)
+
+    PaymentSheetScaffold(
+        topBar = {
+            PaymentSheetTopBar(
+                screen = PaymentSheetScreen.SelectSavedPaymentMethods,
+                showEditMenu = isEditing,
+                isLiveMode = isLiveMode,
+                isProcessing = isProcessing,
+                isEditing = isEditing,
+                handleBackPressed = onBackPressed,
+                toggleEditing = onEdit,
+            )
```

Comment:
I’d love for this to take in `PaymentSheetTopBarState` directly and will take a stab at that next week. Nothing to do in this pull request.